lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -1455,6 +1489,12 @@ SPIRVValue *LLVMToSPIRV::transIntrinsicInst(IntrinsicInst *II,
 }
 
 SPIRVValue *LLVMToSPIRV::transCallInst(CallInst *CI, SPIRVBasicBlock *BB) {
+  if (CI->isIndirectCall())
```

Comment:
I think we need to insert an 'if' here to check if `SPV_INTEL_function_pointers` was enabled